images.md

```diff
@@ -0,0 +1,22 @@
+# Lambda Instance images creation
```

Comment:
The ansible scripts for configuring the system are automatically run by the lambda_instance_manager.py: https://github.com/grnet/okeanos-LoD/blob/devel/core/fokia/lambda_instance_manager.py#L220-L244